feat: updated WACI Issue UI

birtony opened the pull request:
Fixes #1399 

https://user-images.githubusercontent.com/33902374/149846836-4c0b32fd-2c83-4897-af33-85d32f0c2f59.mov

https://user-images.githubusercontent.com/33902374/149846847-bee7405b-f825-4152-b01a-360c7aa8425f.mov

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

sudeshrshetty commented on cmd/wallet-web/src/pages/WACIIssue.vue:
```diff
@@ -4,237 +4,171 @@
     const invitation = toRaw(this.protocolHandler.message());
     const { user, token } = this.getCurrentUser().profile;
-    const credManifest = this.getCredentialManifestData();
+    this.credentialDisplayData = await this.getCredentialManifestData();
```
any reason to put await here? it was there at first place so that this operation can go on asynchronously while we perform DIDComm operation.

At #263 we have await at line #263

birtony replied:
Addressed

sudeshrshetty commented on cmd/wallet-web/src/pages/WACIIssue.vue:
```diff
@@ -253,30 +187,25 @@ export default {
     // get all vaults
     const collectionManager = new CollectionManager({ agent: this.getAgentInstance(), user });
-    this.fetchAllVaults(token, collectionManager);
+    await this.fetchAllVaults(token, collectionManager);
```
fetching all vaults also shouldn't block UI rendering of the page. Did you run into any issue because of that?

birtony replied:
Addressed

sudeshrshetty commented on cmd/wallet-web/src/pages/WACIIssue.vue:
```diff
@@ -329,11 +258,11 @@ export default {
-      this.protocolHandler.done(url);
+      const { status, url } = ack;
+      this.redirectUrl = url;
+      // TODO: issue#1336 - check if status="FAIL", then should redirect to generic error screen, it means WACI flow didn't succeed
```
I see you changed TODO issue number pattern to `issue1336` at one place and `issue#136` another place. We usually follow pattern of `[Issue#<NUMBER>]` format so that it can be easy to locate/search.

birtony replied:
Sure, let's document this format so that everybody is on the same page 👍

sudeshrshetty commented:
nice work @birtony

talwinder50 commented on cmd/wallet-web/src/pages/WACIShare.vue:
```diff
@@ -5,113 +5,134 @@
+          </template>
+        </credential-overview>
+        <!-- List of Credential Banners (Links to Details for each) -->
+        <!-- TODO: issue-1391 -->
```
Do we need to change this issue#1391 ??

birtony commented:
> nice work @birtony 
> 
> 

I've addressed your comments, thanks @sudeshrshetty! I will merge this for now and in case you have any further objections I will raise another issue and address later. Need to deploy this.
